docs/glossary.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Glossary
+sidebar_label: Glossary
+keywords:
+- Harvester
+- glossary
+- terminology
+---
+
+<head>
+  <link rel="canonical" href="https://docs.harvesterhci.io/v1.3/glossary"/>
+</head>
+
+## **guest cluster** / **guest Kubernetes cluster**
+
+Group of integrated Kubernetes worker machines that are running in a VM that is part of a Harvester cluster. Guest clusters form the underlying infrastructure for running containerized workloads.
+
+Creating guest clusters involves pulling images from either the internet or a private registry. You can create guest clusters using both the Harvester UI and the Rancher UI.
+
+## **guest cluster node** / **guest node**
+
+Kubernetes worker machine that uses guest cluster resources to run workloads. 
```

Comment:
Again, a bit unclear. Isn't a guest cluster node a VM running on top of Harvester with this VM then running (part of) the guest K8s cluster as workload?

Reply:
Yup. Maybe says "Kubernetes worker VM"?

Reply:
@m-ildefons @tserong Again, thank you and please check the updated definition.